src/frontend/scss/objects/_form.scss

```diff
@@ -154,11 +175,19 @@
       opacity: 0;
       position: absolute;
       z-index: -1;
+
+      &[disabled] ~ label {
+        cursor: not-allowed;
+
+        span {
+          background-color: r-theme-val(form, input-disabled-background);
+        }
+      }
```

Comment:
Try to target element through class name instead of element to prevent futur issue.

```suggestion
      &[disabled] ~ .form-field__label {
        cursor: not-allowed;

        .form-field__checkbox-control {
          background-color: r-theme-val(form, input-disabled-background);
        }
      }
```